docs/deployments/azure/running-azure-powershell/configuring-the-version-of-the-azure-cli.md

```diff
@@ -3,27 +3,34 @@ title: Configuring the version of the Azure CLI
 description: A version of Azure CLI is bundled with Octopus Deploy and it's possible to configure which version you wish to use in your deployments.
 ---
 
-A version of [Azure CLI](https://docs.microsoft.com/cli/azure/) is bundled with Octopus Deploy.  To determine the version of the bundled CLI, add the command below to an Azure PowerShell Script Step:
+:::warning
+Using the Azure tools bundled with Octopus Deploy is not recommended. Octopus bundles versions of the Azure Resource Manager Powershell modules (AzureRM) and Azure CLI. These were originally provided as convenience mechanisms for users wanting to run scripts against Azure targets. The versions bundled are now out of date, and we will not be updating them further.
+:::
+
+We recommend you configure Octopus Deploy to use your own [version of the Azure CLI](configuring-the-version-of-the-azure-cli.md).
+
+To determine the version of the bundled CLI is installed on your worker, add the command below to an Azure Script Step:
 
 ```powershell
 az --version
 ```
 
-If you wish to use a different version, you can install the Azure CLI on your Octopus Server, and configure Octopus to use the installed version instead. 
+If you wish to use a different version, you can install the Azure CLI on your worker, and configure Octopus to use the installed version.
+
+To be compatible with Octopus's automated Azure CLI login behaviour, the Azure CLI version you install must be 2.0 or above.
```

Comment:
I suppose it is technically possible for a breaking change to be introduced which could introduce future incompatible versions.

This might just need to be something we monitor and update the documentation accordingly.